### PR TITLE
DAOS-623 perf: Restore query test accidentally removed in #5295

### PR DESCRIPTION
**Pull request by jolivier23**
PR #5295 removed both query and rebuild tests from daos_perf
The rebuild test doesn't quite work so it's understandable to
remove it but the query test should not have been removed.

Signed-off-by: Jeff Olivier <jeffrey.v.olivier@intel.com>

### CODE REVIEW

File: src/tests/daos_perf.c [hunk 1]
```diff
@@ -947,6 +999,121 @@ pf_oit(struct pf_test *pf, struct pf_param *param)
 	return rc;
 }
 
+static int
+exclude_server(d_rank_t rank)
+{
+	/** TODO: support exclude */
+	D_ASSERT(0);
```
Comment (jolivier23) [inline src/tests/daos_perf.c:1006]: This needs to be fixed

File: src/tests/daos_perf.c [hunk 2]
```diff
@@ -947,6 +999,121 @@ pf_oit(struct pf_test *pf, struct pf_param *param)
...
+	return 0;
+}
+
+static int
+reint_server(d_rank_t rank)
+{
+	/** TODO: support reintegrate */
+	D_ASSERT(0);
```
Comment (jolivier23) [inline src/tests/daos_perf.c:1014]: This needs to be fixed

File: src/tests/daos_perf.c [hunk 3]
```diff
@@ -1135,6 +1375,18 @@ struct pf_test pf_tests[] = {
 		.ts_parse	= pf_parse_iterate,
 		.ts_func	= pf_iterate,
 	},
+	{
+		.ts_code	= 'R',
+		.ts_name	= "REBUILD",
+		.ts_parse	= pf_parse_rebuild,
+		.ts_func	= pf_rebuild,
+	},
```
Comment (NiuYawei) [inline src/tests/daos_perf.c:1383]: Maybe we'd comment this out before it's ready?

**Review by NiuYawei — DISMISSED** [review]

**Review by NiuYawei — APPROVED** [review]

**Comment by jolivier23** [comment]
@johannlombardi this is the accidental removal I told you about.   The test failures have nothing to do with daos_perf.

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**